src/components/Button/Button.tsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from "react";
+
+interface Props {
+  variant?: string;
+  buttonLabel: string;
+  widthFull?: boolean;
+  children?: JSX.Element;
```

Comment:
JSX.Element es una alternativa para typar `children`, pero tiene algunas limitantes, en cambio `React.ReactNode` nos permite mayores libertades que pueden ser utiles en este componente! Te dejo este link por si quieres verlo mas a detalle: https://stackoverflow.com/questions/58123398/when-to-use-jsx-element-vs-reactnode-vs-reactelement